src/pages/Login/Signup.js

```diff
@@ -0,0 +1,126 @@
+// Import the react JS packages
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { signUp_api } from "../../api/user_api";
+import "./signup.css";
+
+// Define the Login function.
```

Comment:
Don't forget to update comments too when you copy frameworks over, it looks a bit confusing when the comments or logs are incorrect